src/components/organisms/Common/AssetContainer/hooks.ts

```diff
@@ -4,11 +4,11 @@ import { useIntl } from "react-intl";
 
 import {
   GetAssetsQuery,
-  useGetAssetsQuery,
   useCreateAssetMutation,
   useRemoveAssetMutation,
   Maybe,
   AssetSortType as GQLSortType,
+  useGetAssetsQuery,
```

Comment:
Did you move this or your IDE moved this? We have linting on import order and as you can see all the useXXXXQuery imports are grouped together. 
If you moved it, please move it back. If you didn't move it, there might be something wrong with how your IDE is linting so please look into it.